quinn-h3/src/headers.rs

```diff
@@ -0,0 +1,47 @@
+use futures::{Async, Future, Poll};
+use quinn_proto::StreamId;
+
+use crate::{
+    connection::ConnectionRef,
+    proto::{frame::HeadersFrame, headers::Header},
+    Error,
+};
+
+pub struct DecodeHeaders {
+    trailers: Option<HeadersFrame>,
```

Comment:
Can you add some comments on why `DecodeHeaders` needs a `trailers` field? It doesn't seem intuitive to me.

Reply:
This is a naming mistake... Renaming to `frame`. Thanks.